client/my-sites/upgrades/navigation.jsx

```diff
@@ -64,8 +64,8 @@ var NAV_ITEMS = {
 	},
 
 	'My Upgrades': {
-		paths: [ '/my-upgrades' ],
-		label: i18n.translate( 'My Upgrades' ),
+		paths: [ '/purchases' ],
+		label: i18n.translate( 'Manage Purchases' ),
 		external: true
```

Comment:
![screen shot 2015-11-27 at 12 01 29 pm](https://cloud.githubusercontent.com/assets/1130674/11447657/da7b29ce-94fe-11e5-9403-46b280257f1a.png)

`/purchases` is an internal link, so we should try and open this in the window for the same reason that we updated the `target="_blank"` links.

This will require updating [some logic](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/upgrades/navigation.jsx#L163) in `UpgradesNavigation`, because we currently assume that internal links follow the pattern `path + siteSlug`.